docs/en/source/linked-data.md

```diff
@@ -0,0 +1,953 @@
+- JAR\_FILE\_RUN\_SUBMG = SubscriptionManager.jar
+```
+
+# Creating a "Powered by FIWARE" app based on Linked Data
+
+This tutorial recreates the same data entities as the initial _"Powered by FIWARE"_ supermarket finder app, but using
+NGSI-LD linked data entities rather than NGSI v2.
+
+
+## Creating Context Data
+
+When creating linked data entities, it is important to use common data models. This will allow us to easily combine data
+from multiple sources and remove ambiguity when comparing data coming from different sources.
+
+Creating linked data using fully qualified names throughout would be painful, as each attribute would need to be a URI,
+so JSON-LD introduces the idea of an `@context` attribute which can hold pointers to context definitions. To add a
+FIWARE [Building](https://fiware-datamodels.readthedocs.io/en/latest/Building/Building/doc/spec/index.html) data entity,
+the following `@context` would be required
+
+```json
+{
+    "id": "urn:ngsi-ld:Building:store001",
+    "type": "Building",
+    ...  other data attributes
+    "@context": [
+        "https://fiware.github.io/data-models/context.jsonld",
+        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"
+    ]
+}
+```
+
+### Core Context
+
+[https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld](https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld)
+refers to the Core `@context` of NGSI-LD, this defines element such as `id` and `type` which are common to all NGSI
+entities, as well as defining terms such as `Property` and `Relationship`. The core context is so fundamental to
+NGSI-LD, that it is added by default to any `@context` sent to a request.
+
+### FIWARE Data Models
+
+[https://fiware.github.io/data-models/context.jsonld](https://fiware.github.io/data-models/context.jsonld) refers to the
+definition of standard data models supplied by FIWARE. Adding this to the `@context` will load the definitions of all
+the [data models](https://fiwaredata-models.readthedocs.io) defined by the FIWARE Foundation in collaboration with other
+organizations such as [GSMA](https://www.gsma.com/) and [TM Forum](https://www.tmforum.org/). A summary of the FQNs
+related to **Building** can be seen below:
+
+```json
+{
+    "@context": {
+        "Building": "https://uri.fiware.org/ns/data-models#Building",
+        ... etc
+        "address": "http://schema.org/address",
+        "category": "https://uri.fiware.org/ns/data-models#category",
+        "location": "http://uri.etsi.org/ngsi-ld/location",
+        ...etc
+    }
+}
+```
+
+If we include this context definition, it means that we will be able to use short names for `Building`, `address`,
+`location` for our entities, but computers will also be able to read the FQNs when comparing with other sources.
+
+To create a valid **Building** data entity in the context broker, make a POST request to the
+`http://localhost:1026/ngsi-ld/v1/entities` endpoint as shown below. It is essential that the appropriate
```

Comment:
[Major] Please correct the port number as Scorpio query manager run on this port not gateway manager as gateway manager is the interaction point.
It's a generic comment correct it in each document.